migration/migration.go

```diff
@@ -45,7 +45,7 @@ type Migration struct {
 // create the most recent version of the database from scratch.
 var migrations = []Migration{
 	mig1,
-	mig2,
+	mig3,
```

Comment:
Please make sure to add the `mig2`, otherwise you'd just replace my `mig2` with `mig3`.